src/lib/components/Header/Title/Title.svelte

```diff
@@ -3,48 +3,36 @@
 
   import { page } from "$app/stores";
 
-  import Logo from "../Logo";
   import ConditionalWrapper from "$lib/components/ConditionalWrapper";
+  import Logo from "$lib/components/Logo";
 
-  import type { SiteTitleType } from "./types";
-
-  export let siteTitle: SiteTitleType;
+  // We want to provide the site title as the <h1> on the homepage only;
+  //   elsewhere we want to use existing <h1>s and provide accessible link text
+  //   for the logo.
+  $: isHomePage = $page.url.pathname === "/";
+  $: accessibleTitleText = isHomePage
+    ? "Louisiana Department of Agriculture and Forestry"
+    : "Go to home page.";
 </script>
 
-<div class="usa-logo ldaf-logo__wide" id="basic-logo">
-  <a href="/">
-    <Logo />
-  </a>
-  <div class="ldaf-agency-info">
-    <ConditionalWrapper
-      tag="h1"
-      condition={$page.url.pathname === "/"}
-      class="ldaf-header-homepage-heading"
-    >
-      <div class="ldaf-name__wide">
-        <span class="text-bold">{siteTitle.wideTitleRow1}</span>
-        <span class="text-normal">{siteTitle.wideTitleRow2}</span>
-      </div>
-    </ConditionalWrapper>
-    <div class="separator" />
-    <div class="ldaf-commissioner__wide">
-      <span class="text-normal">{siteTitle.commissionerRow1}</span>
-      <span class="text-italic">{siteTitle.commissionerRow2}</span>
```

Comment:
So happy to remove this bit. Never liked the weird edge case this was in terms of how we integrate w/ Contentful

Reply:
Yeah the original idea was coming from the right place (make sure we have all content in Contentful), but the way we set it up was always a half-measure, and I'm still unsure of the best approach for handling one-off strings like this. Much simpler to hard-code (albeit a little more typo-error-prone).